examples/cloud-operations/network-dashboard/cloud-function/main.py

```diff
@@ -0,0 +1,622 @@
+#
```

Comment:
Can you format this file with the same command used here

https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/34f8d05f8dc807467860701967e4e9d1f3d1313b/.github/workflows/linting.yml#L74

We have a check that verifies Python scripts for compliant formatting, and it will fail on this PR unless you format this file.

Reply:
I ran yapf and applied the fixes.